apps/prairielearn/src/pages/instructorCourseAdminSharing/instructorCourseAdminSharing.html.ts

```diff
@@ -131,6 +136,29 @@ export const InstructorSharing = ({
     <html lang="en">
       <head>
         ${renderEjs(__filename, "<%- include('../../pages/partials/head') %>", resLocals)}
+        <script src="${nodeModulesAssetPath('clipboard/dist/clipboard.min.js')}"></script>
+        <script>
+          $(() => {
+            let clipboard = new ClipboardJS('.btn-copy');
+            clipboard.on('success', (e) => {
+              $(e.trigger)
+                .popover({
+                  content: 'Copied!',
+                  placement: 'bottom',
+                })
+                .popover('show');
+              window.setTimeout(function () {
+                $(e.trigger).popover('hide');
+              }, 1000);
+            });
+            $('.js-public-sharing-link-qrcode-button').popover({
+              content: $('#js-public-sharing-link-qrcode'),
+              html: true,
+              trigger: 'click',
+              container: 'body',
+            });
```

Comment:
I copied this from the Assessment settings page, in the interest of making the 'copy this link/qr code' UI look similar. But I'm not sure if that was a good idea. Also, we now have two different copy buttons on this page which look entirely different from one another.

Reply:
I think matching the other copy button on this page would be better. The QR code on other pages is meant for lectures/labs where this will be projected up for students to scan, but I don't think there's currently any use case for that for the public sharing page.

Reply:
QR code removed in 0385fb4; all copy buttons on the page now match.